src/change/controllers/changeController.ts

```diff
@@ -0,0 +1,37 @@
+import { RequestHandler } from 'express';
+import httpStatus from 'http-status-codes';
+import { injectable, inject } from 'tsyringe';
+import { HttpError } from 'express-openapi-validator/dist/framework/types';
+import { Actions } from '@map-colonies/osm-change-generator/dist/models';
+
+import { Services } from '../../common/constants';
+import { ILogger } from '../../common/interfaces';
+import { ChangeManager } from '../models/changeManager';
+import { ChangeModel } from '../models/change';
+import { FlattenedGeoJSON } from '../models/geojsonTypes';
+import { OsmApiElements } from '../models/helpers';
+
+type CreateResourceHandler = RequestHandler<undefined, ChangeModel, ChangeRequestBody>;
+
+export interface ChangeRequestBody {
+  action: Actions;
+  externalId: string;
+  osmElements: OsmApiElements;
+  geojson: FlattenedGeoJSON;
+}
+
+@injectable()
+export class ChangeController {
+  public constructor(@inject(Services.LOGGER) private readonly logger: ILogger, @inject(ChangeManager) private readonly manager: ChangeManager) {}
```

Comment:
same here

Reply:
until a logging strategy will be published, we'll keep it this way